### DISCUSSION TITLE
Can I check when a time series was last updated, or a model was last run

### DISCUSSION THREAD

**Discussion by asmunds** [Volue-Public/energy-mesh-python#478] [Q&A] [answered]
I am wondering if it is possible with energy-mesh-python to check when a time series was last updated, or check when a model was last run.
What I want to do is to make an import of output-data from several cases, and then I want information about when the given cases were last run. Is this possible?

**Comment by tnoczyns-volue** [comment, marked as answer]
Unfortunately such functionality is not available via Mesh Python SDK. In our long-term roadmap we have a functionality for informing clients about changed data via notification mechanism. That could serve your needs to some degree. But we don't have a concrete date for implementing this functionality yet.

**Reply by asmunds** [reply]
Okay, thanks
